FWCore/Integration/test/TestFindProduct.cc

```diff
@@ -29,7 +30,12 @@
 #include <vector>
 
 namespace edmtest {
-  class TestFindProduct : public edm::one::EDAnalyzer<edm::one::WatchRuns, edm::one::WatchLuminosityBlocks> {
+
+  struct DummyTestFindProduct {};
+
+  class TestFindProduct : public edm::one::EDAnalyzer<edm::one::WatchRuns,
+                                                      edm::one::WatchLuminosityBlocks,
+                                                      edm::ProcessBlockCache<DummyTestFindProduct>> {
```

Comment:
Does this illustrate wanting a `edm::one::WatchProcessBlocks` instead of/in addition to `edm::ProcessBlockCache<>`?